src/main/scala/mvp2/actors/InfluxActor.scala

```diff
@@ -4,50 +4,114 @@ import java.net.{InetAddress, InetSocketAddress}
 import akka.actor.Actor
 import com.typesafe.scalalogging.StrictLogging
 import mvp2.messages._
-import mvp2.utils.InfluxSettings
+import scala.concurrent.ExecutionContext.Implicits.global
+import mvp2.utils.{EncodingUtils, InfluxSettings, TestingSettings}
 import org.influxdb.{InfluxDB, InfluxDBFactory}
+import scala.concurrent.duration._
 
-class InfluxActor(settings: InfluxSettings) extends Actor with StrictLogging {
+class InfluxActor(influxSettings: InfluxSettings, testingSettings: Option[TestingSettings]) extends Actor with StrictLogging {
 
   val myNodeAddress: String = InetAddress.getLocalHost.getHostAddress
 
   var pingPongResponsePequestTime: Map[InetSocketAddress, Long] = Map.empty
 
-  val influxDB: InfluxDB = InfluxDBFactory.connect(
-    settings.host,
-    settings.login,
-    settings.password
-  )
+  var msgFromRemote: Map[InetSocketAddress, Map[String, Int]] = Map.empty
+
+  var msgToRemote: Map[InetSocketAddress, Map[String, Int]] = Map.empty
+
+  val port: Int = influxSettings.port
+
+  val influxDB: InfluxDB =
+    InfluxDBFactory.connect(
+      influxSettings.host,
+      influxSettings.login,
+      influxSettings.password
+    )
 
   override def preStart(): Unit = {
-    influxDB.write(settings.port, s"""startMvp value=12""")
+    logger.info("Start influx actor")
+    influxDB.write(port, s"""startMvp value=12""")
+    testingSettings.foreach(testSettings =>
+      context.system.scheduler.schedule(1 seconds, testSettings.iteratorsSyncTime seconds)(syncIterators())
+    )
   }
 
+  def getFromRemoteMsgIncrement(remote: InetSocketAddress, msg: String): Int =
+    msgFromRemote.find(_._1 == remote) match {
+      case Some(msgInfo) => msgInfo._2.find(_._1 == msg) match {
+        case Some(i) =>
+          msgFromRemote = msgFromRemote - msgInfo._1 + msgInfo.copy(_2 = msgInfo._2 - msg + (msg -> (i._2 + 1)))
+          i._2 + 1
+        case None =>
+          msgFromRemote = msgFromRemote - msgInfo._1 + msgInfo.copy(_2 = msgInfo._2 + (msg -> 1))
+          1
+      }
+      case None =>
+        msgFromRemote += (remote -> Map(msg -> 1))
+        1
+    }
+
+  def getToRemoteMsgIncrement(remote: InetSocketAddress, msg: String): Int =
```

Comment:
this and previous function are almost exactly the same. Are you sure, you can't refactor such stuff?